src/ontology/cl-edit.owl

```diff
@@ -4681,8 +4681,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D055098"^^xsd:string) ob
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000163 "FMA:83809"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000163 "endocrinocyte"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000163 "endocrine cell"^^xsd:string)
-EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0060986)))
-SubClassOf(obo:CL_0000163 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368))
+EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000949) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046879)))
```

Comment:
```suggestion
EquivalentClasses(obo:CL_0000163 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046879)))
```

Reply:
lol I changed it to endocrine system from gland due to your comment: 

> In the case of, for example, [endocrine cells of the GI tract](https://pubmed.ncbi.nlm.nih.gov/10524284/), is the intestine considered an 'endocrine gland'?

Reply:
Understandable. Given the complexities, maybe using subclasses instead of an EA is more appropriate for now.